alacritty_terminal/src/term/mod.rs

```diff
@@ -1852,6 +1852,9 @@ impl<T: EventListener> Handler for Term<T> {
     fn clear_line(&mut self, mode: ansi::LineClearMode) {
         trace!("Clearing line: {:?}", mode);
 
+        // Remove active selections
+        self.grid.selection = None;
+
```

Comment:
Just to summarize my comment on IRC, I'd like to see more granularity here.

Clearing everything just because a single line somewhere was cleared seems unfortunate to me.

In our other clear function we could probably be a little more granular too, but that's usually only hit when the entire screen is cleared.

Reply:
I agree, I don't like this "fix" either. For example it'll break tmux. We also have a lot of ways to clear content underneath the selection. In theory we can only clear selection in mouse mode when left clicking, since mouse mode app could have its own selection(tmux) and user can have both selections with the same color, so it could be very confusing.